packages/components/src/mobile/notification-sheet/index.native.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import { Platform, View, Text } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { BottomSheet, Icon } from '@wordpress/components';
+import { withPreferredColorScheme } from '@wordpress/compose';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import styles from './style.scss';
+
+function NotificationSheet( { title, getStylesFromColorScheme, isVisible, onClose, type = 'singular', withMessage = true, withQuotes = true } ) {
```

Comment:
To me "notification" implies a more generic component to show information to the user. If this is just about the sheet shown when there's a feature that's not available yet, I'd go for something like `UnsupportedFeatureNotice` or something like that.

Reply:
Thanks for the suggestion, looks good to me!

Reply:
Outdated - `NotifcationSheet` is removed since we are not using bottom sheet to inform user about missing theme colors.